config/locales/en.yml

```diff
@@ -1367,11 +1367,12 @@ en:
       community_driven_title: Community Driven
       community_driven_html: |
         OpenStreetMap's community is diverse, passionate, and growing every day.
-        Our contributors include enthusiast mappers, GIS professionals, engineers
+        Our contributors include people like you: enthusiast mappers, GIS professionals, engineers
         running the OSM servers, humanitarians mapping disaster-affected areas,
         and many more.
-        To learn more about the community, see the
-        <a href='https://blog.openstreetmap.org'>OpenStreetMap Blog</a>,
+        To learn more about the community, see the OpenStreetMap
+        <a href='https://welcome.openstreetmap.org/'>Welcome Mat</a>,
```

Comment:
The "welcome mat" is a page designed specifically for an organisational audience, and is not aimed at individual contributors.

So if we're aiming this text at "people like you: <list of individuals>" then I think we're mixing up our messaging by having the welcome mat as the first link. I agree that the first link should cover similar ground to some of the topics shown on the welcome mat, but it should be written for an audience of individuals. I'm open to suggestions for what content is already out there which fills that purpose.